data_steward/cdr_cleaner/cleaning_rules/suppress_combined_pfmh_survey.py

```diff
@@ -0,0 +1,553 @@
+"""
+Removing data associated with the combined Personal Family Health Survey.
+
+This survey is expected to be included in the November CDR and later.  We
+need to be ready to remove this data from the May CDR.
+
+Original Issue:  DC-2146
+"""
+# Python Imports
+import logging
+
+# Third party imports
+from jinja2 import Template
+
+# Project imports
+from common import OBSERVATION
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+
+LOGGER = logging.getLogger(__name__)
+
+ISSUE_NUMBERS = ['DC2146']
+
+# Save rows that will be dropped to a sandboxed dataset.
+DROP_SELECTION_QUERY_TMPL = Template("""
+CREATE OR REPLACE TABLE `{{project}}.{{sandbox}}.{{drop_table}}` AS
+SELECT
+  *
+FROM
+  `{{project}}.{{dataset}}.observation`
+WHERE
+  -- separated surveys were removed from participant portal on 2021-11-01 --
+  -- combined survey was launced on 2021-11-01 --
+  observation_date > '2021-10-31'
+  AND LOWER(observation_source_value) IN ('record_id',
```

Comment:
suggestion: I think it is better to store this list of concepts as a constant or in a separate file, since we will be adding a check in the rdr_qc notebook, we can reuse the list instead of copying this list of 100s of concepts and maintain these in multiple locations.

Reply:
I pulled the list into a separate variable.  I think a notebook should be able to import and use the list.  I was hoping to avoid creating another file since this is likely a one time fix.